examples/compute_polar_coordinates.py

```diff
@@ -0,0 +1,362 @@
+
+# %%
+# Histogram of ``phi`` values
+# -------------------------------------
+# We can also explore which ``phi`` values are most common in the dataset with
+# a circular histogram.
+
+# compute number of bins
+bin_width_deg = 5  # width of the bins in degrees
+n_bins = int(360 / bin_width_deg)
+
+# remove NaN values from the phi data
+bool_not_nan = (
+    ~np.isnan(
+        head_vector_polar.sel(individuals=mouse_name, space_pol="phi").values
+    ),
+)
+
+# compute histogram
+counts, bins = np.histogram(
+    head_vector_polar.sel(individuals=mouse_name, space_pol="phi").values[
+        bool_not_nan
+    ],
+    bins=np.linspace(-np.pi, np.pi, n_bins + 1),
+)
+
+
+# plot histogram as a bar plot in polar projection
+fig = plt.figure()
+ax = fig.add_subplot(projection="polar")
+
+bin_width_rad = np.deg2rad(bin_width_deg)
+bars = ax.bar(bins[:-1], counts, width=bin_width_rad, align="edge")
+
+ax.set_title("phi histogram")
+ax.set_theta_direction(-1)  # set direction clockwise
+ax.set_theta_offset(0)  # set zero at the right
+
+# set xticks to match the phi values in degrees
+n_xtick_edges = 9
+ax.set_xticks(np.linspace(0, 2 * np.pi, n_xtick_edges)[:-1])
+xticks_in_deg = (
+    list(range(0, 180 + 45, 45)) + list(range(0, -180, -45))[-1:0:-1]
+)
+ax.set_xticklabels([str(t) + "\N{DEGREE SIGN}" for t in xticks_in_deg])
+
+fig.show()
+
+# %%
+# The ``phi`` circular histogram shows that the head vector appears at a
+# variety of orientations in this dataset.
+
+# %%
+# Polar plot of the head vector within a time window
+# ---------------------------------------------------
+# We can also use a polar plot to represent the head vector in time,
+# in a coordinate system centred at the head of the mouse. Again,
+# this will be easier to visualise if we focus on a smaller time window.
+
+# select phi values within a time window
+phi = head_vector_polar.sel(
+    individuals=mouse_name,
+    space_pol="phi",
+    time=time_window,
+).values
+
+# plot tip of the head vector within that window, and color based on time
+fig = plt.figure()
+ax = fig.add_subplot(projection="polar")
+sc = ax.scatter(
+    phi,
+    np.ones_like(phi),
+    c=time_window,
+    cmap="viridis",
+    s=50,
+)
+ax.set_theta_direction(-1)  # set direction counterclockwise
+ax.set_theta_offset(0)  # set zero at the right
+cax = fig.colorbar(
+    sc,
+    ax=ax,
+    label=f"time ({ds.attrs['time_unit']})",
+    ticks=list(time_window)[0::2],
+)
+
+# set xticks to match the phi values in degrees
+n_xtick_edges = 9
+ax.set_xticks(np.linspace(0, 2 * np.pi, n_xtick_edges)[:-1])
+xticks_in_deg = (
+    list(range(0, 180 + 45, 45)) + list(range(0, -180, -45))[-1:0:-1]
+)
+ax.set_xticklabels([str(t) + "\N{DEGREE SIGN}" for t in xticks_in_deg])
+
+fig.show()
+
+# %%
+# The polar plot shows how in this small time window of 20 frames,
+# the head of the mouse moved from right to left.
+
+# %%
+# ``movement`` also provides a ``pol2cart`` convenience function to transform
```

Comment:
This bit should be under a separate subheading (since it doesn't belong to the "polar plot"). Alternatively you can also mention this earlier (e.g. when you first do the cart2po), up to you.